appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -756,8 +758,10 @@ private YaVersion() {
   // - The BluetoothClient.Secure property was added.
   // For BLUETOOTHCLIENT_COMPONENT_VERSION 6:
   // - The BluetoothClient.DisconnectOnError property was added.
-  // For BLUETOOTH_CLIENT_COMPONENT_VERSION 7:
-  // - The BluuetoothClient.PollingRate property was added.
+  // For BLUETOOTHCLIENT_COMPONENT_VERSION 7:
+  // - The BluetoothClient.PollingRate property was added.
+  // For BLUETOOTHCLIENT_COMPONENT_VERSION 8:
+  // - The BluetoothClient.NoLocationNeeded property was added.
   public static final int BLUETOOTHCLIENT_COMPONENT_VERSION = 7;
```

Comment:
Oops, forgot to increment this to 8!
Don't worry, I caught and fixed this during the merge (had a conflict to deal with)